src/ui/osx/TogglDesktop/Feature/Permission/ObjcSystemPermissionManager.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ObjcSystemPermissionManager.swift
+//  TogglDesktop
+//
+//  Created by Nghia Tran on 11/15/19.
+//  Copyright © 2019 Alari. All rights reserved.
+//
+
+import Foundation
+
+// Objc bridge
+@objc final class ObjcSystemPermissionManager: NSObject {
```

Comment:
It's a helper class to expose the Swift class to objc without messing up the original code.